.github/workflows/build-push-images.yaml

```diff
@@ -220,4 +232,84 @@ jobs:
           password: ${{ secrets.QUAY_TOKEN }}
       - name: Print image url
         if: steps.filter.outputs.vulnerability == 'true'
-        run: echo "Image pushed to ${{ steps.push-to-quay-vulnerability-scan.outputs.registry-paths }}"
+        run: |
+          echo "Image pushed to ${{ steps.push-to-quay-vulnerability-scan.outputs.registry-paths }}"
+          echo "::set-output name=vulnerability_scan_pushed::true"
```

Comment:
Is the second output var (example `steps.build-image-access-setup.outputs.vulnerability_scan_pushed`) necessary? 
Can't we reuse `steps.filter.outputs.vulnerability` ?
(same for the rest of the builds)

Reply:
Because, unless the path is changed for a particular image and the step is executed, steps.filter.outputs.vulnerability will be non-existent.

Reply:
let's take this example
```
- name: Print image url
        if: steps.filter.outputs.vulnerability == 'true'
        run: |
          echo "Image pushed to ${{ steps.push-to-quay-vulnerability-scan.outputs.registry-paths }}"
          echo "::set-output name=vulnerability_scan_pushed::true"
```
If `steps.filter.outputs.vulnerability` does not exist (path did not change for this image) then `vulnerability_scan_pushed` won't exist either no ?

Reply:
Addressed in the other comment. Resolving this one to avoid multiple threads.